README.md

```diff
@@ -26,4 +26,6 @@ So in this task requirements are next:
   You should use environment variables for that purpose. So, read `API_KEY` from environment. 
   And to pass environment variables to docker container use `-e` flag.
 - You must *modify* this line with correct command to pull your image:
+  I have no idea, just because I can't run this staff. so I'm thinking something like:
+  COMMAND=`docker pull <magus/weather>`.
```

Comment:
you can find the correct command on dockerhub. Delete redundant text from readme
<img width="1373" alt="image" src="https://user-images.githubusercontent.com/47711774/181236935-2ce14562-b1d7-4401-962d-f013e7d0f850.png">